src/sp_disabled_functions.c

```diff
@@ -141,8 +147,18 @@ static bool is_param_matching(zend_execute_data* execute_data,
 
       /* This is the parameter name we're looking for. */
       if (true == pcre_matching || config_node->pos != -1) {
+        // Use a local zval to not have to alloc one.
         arg_value = ZEND_CALL_ARG(execute_data, i + 1);
 
+        if (is_variadic) {
+          // This is where the fun begins.
+          // PHP copy the extra arguments in `zend_copy_extra_args` using this delta.
+          // The current argument is never used again.
+          size_t delta = (execute_data->func->op_array.last_var
```

Comment:
Wow, this is ugly as sin! PHP does it the same way?

Reply:
[Yes](https://github.com/php/php-src/blob/master/Zend/zend_execute.c#L2424), I have no idea why :>

Reply:
Ugh.